src/routes/+page.server.ts

```diff
@@ -0,0 +1,20 @@
+import { lucia } from "$lib/server/auth";
+import { fail, redirect, type Actions, type ServerLoad} from "@sveltejs/kit";
+export const load: ServerLoad = async ({ locals }) => {
+	// ...
+};
+
+export const actions: Actions = {
+	default: async (event) => {
+		if (!event.locals.session) {
+			return fail(401);
+		}
+		await lucia.invalidateSession(event.locals.session.id);
+		const sessionCookie = lucia.createBlankSessionCookie();
+		event.cookies.set(sessionCookie.name, sessionCookie.value, {
+			path: ".",
+			...sessionCookie.attributes
+		});
```

Comment:
I believe that is what invalidateSession does?

Reply:
lucia.invalidateSession only updates the DB not the event.locals. It doesn't really matter because we are redirecting anyways (which clears the event.locals) but it's good practice.